src/utils/media-utils.ts

```diff
@@ -0,0 +1,51 @@
+import axios from "axios"
+
+import { config } from "@config/config"
+
+import logger from "@root/logger/logger"
+import { MediaFileInput, MediaFileOutput } from "@root/types"
+import { getAccessToken } from "@root/utils/token-retrieval-utils"
+
+const GITHUB_ORG_NAME = config.get("github.orgName")
+
+export const isMediaFileOutput = (t: MediaFileOutput): t is MediaFileOutput =>
```

Comment:
suggest changing the input type to `unknown`. because this function's precondition and postcondition is identical, we are asserting something **we already know** to use it. also realised this has 0 refs, so we can also delete it ._.

Reply:
This was originally used for the tests when the output type was variable - since we've moved out the directory functionality of this method, this is no longer necessary! removed in https://github.com/isomerpages/isomercms-backend/pull/740/commits/7e317a77d5238c6f9f1ce57bcf4055f4cd531275